website/components/AnimatedLogo/index.js

```diff
@@ -0,0 +1,39 @@
+"use strict";
+
+const React = require("react");
+const h = React.createElement;
+const data = require("./data.json");
+
+const Line = props =>
+  h(
+    "g",
+    { className: `l${props.index}` },
+    props.dashes.map(({ color }, j) =>
+      h("path", {
+        key: j,
+        d: `m 5 ${props.index * 20 + 5} l ${props.total} 0`,
+        className: ["dash", `p${j}`, `c${color}`].join(" ")
+      })
+    )
+  );
+
```

Comment:
Why not use JSX here?

Reply:
For some reason docusaurus wasn't preprocessing jsx on it. Forgot to raise a bug.